x-pack/plugins/ml/public/application/components/feedback_button/feedback_button.tsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState, useEffect, FC, useMemo } from 'react';
+import { MlPages, ML_PAGES } from '../../../locator';
+import { useMlKibana } from '../../contexts/kibana';
+import { useEnabledFeatures } from '../../contexts/ml';
+
+import { useJobsApiService } from '../../services/ml_api_service/jobs';
+import { useCloudCheck } from '../node_available_warning/hooks';
+import { FeatureFeedbackButton } from './feature_feedback_button';
+
+interface Props {
+  jobIds: string[];
+  page: MlPages;
+}
+
+const FORM_IDS = {
+  SINGLE_METRIC_VIEWER: '1FAIpQLSdlMYe3wuJh2KtBLajI4EVoUljAhGjJwjZI7zUY_Kn_Sr2lug',
+  ANOMALY_EXPLORER: '1FAIpQLSfF1Ry561b4lYrY7iiyXhuZpxFzAmy2c9BFUT3J2AJUevY1iw',
+};
+
+const MATCHED_CREATED_BY_TAGS = ['ml-module-metrics-ui-hosts'];
+
+export const FeedBackButton: FC<Props> = ({ jobIds, page }) => {
+  const { jobs: getJobs } = useJobsApiService();
+  const {
+    services: { kibanaVersion },
+  } = useMlKibana();
+  const { isCloud } = useCloudCheck();
+  // ML does not have an explicit isServerless flag,
+  // it does however have individual feature flags which are set depending
+  // whether the environment is serverless or not.
+  // showNodeInfo will always be false in a serverless environment
+  // and true in a non-serverless environment.
+  const { showNodeInfo } = useEnabledFeatures();
+
+  const [jobIdsString, setJobIdsString] = useState<string | null>(null);
+  const [showButton, setShowButton] = useState(false);
+
+  const formId = useMemo(() => getFormId(page), [page]);
+
+  useEffect(() => {
+    const tempJobIdsString = jobIds.join(',');
+    if (tempJobIdsString === jobIdsString || tempJobIdsString === '') {
+      return;
+    }
+    setJobIdsString(tempJobIdsString);
+
+    getJobs(jobIds).then((resp) => {
+      setShowButton(false);
+      for (const job of resp) {
+        const createdBy = job.custom_settings?.created_by;
+
+        if (createdBy && MATCHED_CREATED_BY_TAGS.includes(createdBy)) {
+          setShowButton(true);
+          break;
+        }
+      }
```

Comment:
```suggestion
      setShowButton(resp.some(job => MATCHED_CREATED_BY_TAGS.includes(job.custom_settings?.created_by)))
```

Reply:
Updated in https://github.com/elastic/kibana/pull/175613/commits/ba2c7e8c9faf81f0810a078f53563c85ab682954